pw-pbkdf2.c

```diff
@@ -229,6 +246,95 @@ static int pbkdf2_encrypt(
 						  dk.bv_len, iteration,
 						  salt.bv_len, (const uint8_t *) salt.bv_val,
 						  dk.bv_len, dk_value);
+#elif HAVE_MOZNSS
+	SECStatus rv;
+	rv = NSS_NoDB_Init(".");
+	if (rv != SECSuccess)
+ 	{
+    		fprintf(stderr, "NSS initialization failed (err %d)\n",
+        	PR_GetError());
+  	}
+        PK11SlotInfo *slot = NULL;
+	SECAlgorithmID * algid = NULL;
+	PRArenaPool *arena1;
+	PRArenaPool *arena2;
+	PRArenaPool *arena3;
+	arena1 = PORT_NewArena(DER_DEFAULT_CHUNKSIZE);
+	arena2 = PORT_NewArena(DER_DEFAULT_CHUNKSIZE);
+	arena3 = PORT_NewArena(DER_DEFAULT_CHUNKSIZE);
```

Comment:
Please lease perform suitable naming. ex: xxx_arena, arena_xxx.
BTW,, are these three independent arenas required?

Reply:
ok about the naming..not sure if they are all needed